TESTING.md

```diff
@@ -400,7 +400,7 @@ available. Click here to open the feedback.** notification is shown.
 
 **12.1** Click **Filter Assignments** button (pink funnel icon) above the assignments tree.
 
-**12.2** ASSERTION: **Non-submittable** menu item is checked.
+**12.2** ASSERTION: **Non-submittable** and **Completed** menu items are checked.
```

Comment:
This can be a bit difficult to test because tester doesn't necessarily have any completed assignments. Thus, I'd leave this away from the TESTING.md